packages/svelte-check/src/index.ts

```diff
@@ -75,6 +76,12 @@ async function getDiagnostics(
 
             let fileHasProblems = false;
 
+            if(!diagnostic.diagnostics.length) {
```

Comment:
`diagnostic.diagnostics.length` is the diagnostics count of each file, not the files count. Also, you need to run `yarn format`